pkgs/by-name/sy/syspower/package.nix

```
@@ -0,0 +1,47 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  makeWrapper,
+  pkg-config,
+  gtkmm4,
+  gtk4-layer-shell,
+  ...
+}:
+stdenv.mkDerivation {
+  pname = "syspower";
+  version = "0-unstable-2024-12-10";
+
+  src = fetchFromGitHub {
+    owner = "System64fumo";
+    repo = "syspower";
+    rev = "b205c3443daaee6c3454f23b93b78f42c10221a1";
+    hash = "sha256-kLaWIaMTRurY7a8Nn7C9ft/Zdvx35AiU7Db8n/NFy90=";
+  };
+
+  buildInputs = [
+    gtkmm4
+    gtk4-layer-shell
+  ];
+
+  nativeBuildInputs = [
+    makeWrapper
+    pkg-config
+  ];
+
+  installPhase = ''
+    install -D -t $out/bin syspower
+    install -D -t $out/lib libsyspower.so
+    install -D -t $out/share/sys64/power config.conf style.css
+    wrapProgram $out/bin/syspower --prefix LD_LIBRARY_PATH : $out/lib
+  '';
```

```suggestion
  makeFlags = [ "PREFIX=${placeholder "out"}" ];

  postFixup = ''
    wrapProgram $out/bin/syspower --prefix LD_LIBRARY_PATH : $out/lib
  '';
```

Upstream provides a `make install` target mkDerivation will automatically use if we help it

Wrapping programs also generally goes into the fixupPhases